### ISSUE TITLE
WebUI only shows unresponsive message "Confirm the character deletion?"

### ISSUE THREAD

**Issue by allo-** [oobabooga/text-generation-webui#4422]
### Describe the bug

Since I updated the webui, I only get a seemingly broken message "Confirm the character deletion?" when accessing the webinterface. The message is centered, but the buttons "Delete" and "Cancel" are at the upper left corner of the page. The buttons do nothing and there is no way to close the dialog or what this should be to access the webui.

Commit where it happens: 262f8ae5bb49b2fb1d9aac9af01e3e5cd98765db

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

1. Run python server.py
2. Access the webui

### Screenshot

![grafik](https://github.com/oobabooga/text-generation-webui/assets/1482341/0c2b2bbd-8792-4548-b63a-5b4474611525)

### Logs

```shell
2023-10-30 21:03:03 INFO:Loading the extension "gallery"...
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
```

### System Info

```shell
OS: Debian trixie.
Python: 3.10.13
```

**Comment by remixer-dec** [comment]
Caused by https://github.com/oobabooga/text-generation-webui/commit/fae8062d393edb84f74ec3ad5a1192dd83eb6875

**Comment by socksy** [comment]
I had this bug, but fixed it by running `./update_macos.sh` (on macOS, obviously)

**Comment by allo-** [comment]
Indeed, updating dependencies with `pip install -r requirement.txt` helped. Can this bug be closed or is there still things you want to do about it?

**Comment by allo-** [comment]
This one is indeed no longer relevant.
